src/run.js

```diff
@@ -17,7 +17,7 @@ export const run = (element = window.document, initAttr = 'data-nc') =>
         // get the component that needs, will load by attribute
```

Comment:
Minor: Not sure if it's better to have observer here, when we have the node, instead in a factory.
Since in factory observer is called (created and destroyed) for each Component (data-nc="Com1, Com2, Com3"), while we need it only for element, regardless of Components.
This is not used that much, but still it's an option.